Translation/en/RedirectedResources/assets/abdata/communication/info_00/tips_00/translation.txt

```diff
@@ -1,19 +1,19 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.2 (KoikatsuSunshine) by KKS_TextDump v1.4.5.1
 //
-//ヤマネコ >=
-//<color%3D#ffa801ff>ヤマネコ</color>を<color%3D#ffa801ff>自然公園</color>や<color%3D#ffa801ff>路地</color>でたまーに見かけるって聞くよ。見つけられたらラッキー！=
-//ヤシガニ >=
-//<color%3D#ffa801ff>浜辺</color>や<color%3D#ffa801ff>自然公園</color>には稀に<color%3D#ffa801ff>ヤシガニ</color>が出る事があるよ。立派なハサミで挟まれないように注意！=
-//ロマンチックな告白は…… >=
-//<color%3D#ffa801ff>流れ星が綺麗に見える</color>ような静かな場所で愛の<color%3D#ffa801ff>告白</color>……ロマンチックな告白に憧れる～！=
-//もう一つのビーチ？ >=
-//この島には立ち入り禁止になってる<color%3D#ffa801ff>もう一つのビーチ</color>があるんだよ。今はどうなってるかなぁ～？=
-//ウムイの鐘 >=
-//<color%3D#ffa801ff>灯台</color>の近くに<color%3D#ffa801ff>ウムイの鐘</color>がおいてあるよ。<color%3D#ffa801ff>恋人同士</color>でその鐘を鳴らすと、その二人は必ず結ばれるっていう噂があるんだ～。=
-//カップルドリンク >=
-//<color%3D#ffa801ff>ホテルのプールサイド</color>で注文できる<color%3D#ffa801ff>トロピカルドリンク</color>が人気だよ。<color%3D#ffa801ff>恋人同士</color>で一緒に飲むのがブームみたい。=
-//デートスポット >=
-//この島には<color%3D#ffa801ff>カップル</color>で楽しめる<color%3D#ffa801ff>スポット</color>がいくつかあるんだよ。そういう相手がいるなら一緒に周ってみてね！=
-//バナナボート >=
-//<color%3D#ffa801ff>浜辺のレンタルショップ</color>で頼めば<color%3D#ffa801ff>バナナボート</color>を体験できるよ。よかったら<color%3D#ffa801ff>恋人同士</color>で一緒に体験してみてね！=
+ヤマネコ >=Wildcat
+<color%3D#ffa801ff>ヤマネコ</color>を<color%3D#ffa801ff>自然公園</color>や<color%3D#ffa801ff>路地</color>でたまーに見かけるって聞くよ。見つけられたらラッキー！=I hear that <color%3D#ffa801ff>wildcats</color> are sometimes seen in <color%3D#ffa801ff>nature parks</color> and <color%3D#ffa801ff>alleyways</color>. You're lucky if you can find one!
+ヤシガニ >=Coconut Crabs
+<color%3D#ffa801ff>浜辺</color>や<color%3D#ffa801ff>自然公園</color>には稀に<color%3D#ffa801ff>ヤシガニ</color>が出る事があるよ。立派なハサミで挟まれないように注意！=In <color%3D#ffa801ff>beaches</color> and <color%3D#ffa801ff>nature parks</color>, <color%3D#ffa801ff>coconut crabs</color> rarely appear. Be careful not to get pinched by their magnificent claws!
+ロマンチックな告白は…… >=Confessions
+<color%3D#ffa801ff>流れ星が綺麗に見える</color>ような静かな場所で愛の<color%3D#ffa801ff>告白</color>……ロマンチックな告白に憧れる～！=I long for a <color%3D#ffa801ff>romantic confession of love</color> in a quiet place where I can <color%3D#ffa801ff>see shooting stars</color>......
+もう一つのビーチ？ >=Another Beach?
+この島には立ち入り禁止になってる<color%3D#ffa801ff>もう一つのビーチ</color>があるんだよ。今はどうなってるかなぁ～？= There's <color%3D#ffa801ff>another beach</color> on the island that's off-limits. I wonder what's happening there right now~?
```

Comment:
```suggestion
この島には立ち入り禁止になってる<color%3D#ffa801ff>もう一つのビーチ</color>があるんだよ。今はどうなってるかなぁ～？= There's <color%3D#ffa801ff>another beach</color> on the island that's off-limits. I wonder what's it's like there now~?
```
Question is about the state of the beach after all this time (what's changed/if it looks different), not what's happening at this moment.